src/Client.ts

```diff
@@ -220,6 +220,7 @@ export default class Client {
 
       const responseJson: ResponseBody = JSON.parse(responseText)
       this.log(LogLevel.INFO, `request success`, { method, path })
+      this.log(LogLevel.DEBUG, `succeed response body`, { body: responseText })
```

Comment:
could you make this consistent with the previous log?
```suggestion
      this.log(LogLevel.DEBUG, `response body success`, { body: responseText })
```

Reply:
Hello @varunrau! Thank you for the comment.

I believe your opinion is appropriate, but I have added it to align with the pattern seen in the after log, especially considering the 230-241 lines handling errors.

When an error occurs in the request
 * request fail -> failed response body

When the request is succeeded
 * request success -> **succeeded response body** (what I added)